solutionbox/structured_data/datalab_solutions/structured_data/_package.py

```diff
@@ -447,20 +471,12 @@ def cloud_predict(model_name, model_version, data, is_target_missing=False):
   Args:
     model_name: deployed model name
     model_verion: depoyed model version
-    data: List of csv strings that match the model schema. Or a pandas DataFrame 
-        where the columns match the model schema. The first column,
-        the target column, is assumed to exist in the data. 
-    is_target_missing: If true, prepends a ',' in each csv string or adds an 
-        empty DataFrame column. If the csv data has a leading ',' keep this flag
-        False. Example: 
-        1) If data = ['target,input1,input2'], then set is_target_missing=False.
-        2) If data = [',input1,input2'], then set is_target_missing=False.
-        3) If data = ['input1,input2'], then set is_target_missing=True.
+    data: List of csv strings or a Pandas DataFrame that match the model schema.
 
   Before using this, the model must be created. This can be done by running
   two gcloud commands:
   1) gcloud beta ml models create NAME
-  2) gcloud beta ml models versions create VERSION --model NAME \
+  2) gcloud beta ml versions create VERSION --model NAME \
       --origin gs://BUCKET/training_output_dir/model
   or one datalab magic:
   1) %mlalpha deploy --name=NAME.VERSION \
```

Comment:
update that command. It should be "ModelVersions('mymodel').deploy('myversion')".